### PR TITLE
updating compute schemas 

### PR DESCRIPTION
**Pull request by grizzlytheodore**
updating compute schemas. 

we plan on on-boarding for auto-generation, but for time being, manual update.

### CODE REVIEW

**Review by anthony-c-martin — CHANGES_REQUESTED** [review]
Please add $refs to the new resources to `schemas/2019-04-01/deploymentTemplate.json`

**Comment by anthony-c-martin** [comment]
You need to replace any instances of `https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression` with `https://schema.management.azure.com/schemas/common/definitions.json#/definitions/expression` to fix the validation. Do you know where the former came from?

File: schemas/2020-12-01/Microsoft.Compute.Extensions.json [hunk 1]
```diff
@@ -0,0 +1,2321 @@
...
+          "minLength": 1,
+          "description": "Microsoft.Compute/extensions - Type handler version"
+        },
+        "settings": {
+          "oneOf": [
+            {
+              "type": "object"
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
```
Comment (grizzlytheodore) [inline schemas/2020-12-01/Microsoft.Compute.Extensions.json:327]: right here for some reason

**Comment by grizzlytheodore** [comment]
addressed your comment @anthony-c-martin

**Comment by grizzlytheodore** [comment]
@anthony-c-martin  how can i address these errors? 

2021-02-25T16:15:26.4924733Z   3 failing
2021-02-25T16:15:26.4925254Z 
2021-02-25T16:15:26.4945949Z   1) Validate test templates against VSCode language service
2021-02-25T16:15:26.4948020Z        running schema validation on '/home/runner/work/azure-resource-manager-schemas/azure-resource-manager-schemas/tools/templateTests/Template_1.2019-04-01.json':
2021-02-25T16:15:26.4949156Z 
2021-02-25T16:15:26.4949746Z       AssertionError: expected [ Array(1) ] to deeply equal []
2021-02-25T16:15:26.4950457Z       + expected - actual
2021-02-25T16:15:26.4950734Z 
2021-02-25T16:15:26.4951122Z       -[
2021-02-25T16:15:26.4951667Z       -  {
2021-02-25T16:15:26.4952113Z       -    "code": 768
2021-02-25T16:15:26.4953693Z       -    "message": "$ref '/definitions/expression' in 'https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json' can not be resolved."
2021-02-25T16:15:26.4955035Z       -    "range": {
2021-02-25T16:15:26.4955649Z       -      "end": {
2021-02-25T16:15:26.4956264Z       -        "character": 94
2021-02-25T16:15:26.4956877Z       -        "line": 1
2021-02-25T16:15:26.4957409Z       -      }
2021-02-25T16:15:26.4957990Z       -      "start": {
2021-02-25T16:15:26.5008450Z       -        "character": 13
2021-02-25T16:15:26.5009234Z       -        "line": 1
2021-02-25T16:15:26.5010069Z       -      }
2021-02-25T16:15:26.5011123Z       -    }
2021-02-25T16:15:26.5011718Z       -    "severity": 2
2021-02-25T16:15:26.5066124Z       -  }
2021-02-25T16:15:26.5066627Z       -]
2021-02-25T16:15:26.5067242Z       +[]
2021-02-25T16:15:26.5067662Z       
2021-02-25T16:15:26.5068213Z       at Context.<anonymous> (tests.ts:226:30)
2021-02-25T16:15:26.5068659Z 
2021-02-25T16:15:26.5069396Z   2) Validate test templates against VSCode language service
2021-02-25T16:15:26.5071245Z        running schema validation on '/home/runner/work/azure-resource-manager-schemas/azure-resource-manager-schemas/tools/templateTests/Template_2.2019-04-01.json':
2021-02-25T16:15:26.5072385Z 
2021-02-25T16:15:26.5073046Z       AssertionError: expected [ Array(1) ] to deeply equal []
2021-02-25T16:15:26.5073854Z       + expected - actual
2021-02-25T16:15:26.5074241Z 
2021-02-25T16:15:26.5074742Z       -[
2021-02-25T16:15:26.5075275Z       -  {
2021-02-25T16:15:26.5075843Z       -    "code": 768
2021-02-25T16:15:26.5077439Z       -    "message": "$ref '/definitions/expression' in 'https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json' can not be resolved."
2021-02-25T16:15:26.5078790Z       -    "range": {
2021-02-25T16:15:26.5079386Z       -      "end": {
2021-02-25T16:15:26.5080010Z       -        "character": 94
2021-02-25T16:15:26.5080635Z       -        "line": 1
2021-02-25T16:15:26.5081187Z       -      }
2021-02-25T16:15:26.5081846Z       -      "start": {
2021-02-25T16:15:26.5083957Z       -        "character": 13
2021-02-25T16:15:26.5084524Z       -        "line": 1
2021-02-25T16:15:26.5084950Z       -      }
2021-02-25T16:15:26.5085638Z       -    }
2021-02-25T16:15:26.5086096Z       -    "severity": 2
2021-02-25T16:15:26.5086547Z       -  }
2021-02-25T16:15:26.5086931Z       -]
2021-02-25T16:15:26.5087218Z       +[]
2021-02-25T16:15:26.5087513Z       
2021-02-25T16:15:26.5087950Z       at Context.<anonymous> (tests.ts:226:30)
2021-02-25T16:15:26.5088273Z 
2021-02-25T16:15:26.5088804Z   3) Validate test templates against VSCode language service
2021-02-25T16:15:26.5090471Z        running schema validation on '/home/runner/work/azure-resource-manager-schemas/azure-resource-manager-schemas/tools/templateTests/Template_3.2019-04-01.json':
2021-02-25T16:15:26.5091480Z 
2021-02-25T16:15:26.5092001Z       AssertionError: expected [ Array(1) ] to deeply equal []
2021-02-25T16:15:26.5092685Z       + expected - actual
2021-02-25T16:15:26.5092956Z 
2021-02-25T16:15:26.5093331Z       -[
2021-02-25T16:15:26.5093716Z       -  {
2021-02-25T16:15:26.5094155Z       -    "code": 768
2021-02-25T16:15:26.5095681Z       -    "message": "$ref '/definitions/expression' in 'https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json' can not be resolved."
2021-02-25T16:15:26.5096926Z       -    "range": {
2021-02-25T16:15:26.5097377Z       -      "end": {
2021-02-25T16:15:26.5097888Z       -        "character": 94
2021-02-25T16:15:26.5098366Z       -        "line": 1
2021-02-25T16:15:26.5098808Z       -      }
2021-02-25T16:15:26.5099244Z       -      "start": {
2021-02-25T16:15:26.5099976Z       -        "character": 13
2021-02-25T16:15:26.5100476Z       -        "line": 1
2021-02-25T16:15:26.5100896Z       -      }
2021-02-25T16:15:26.5101314Z       -    }
2021-02-25T16:15:26.5101770Z       -    "severity": 2
2021-02-25T16:15:26.5102212Z       -  }
2021-02-25T16:15:26.5102601Z       -]
2021-02-25T16:15:26.5102885Z       +[]
2021-02-25T16:15:26.5103175Z       
2021-02-25T16:15:26.5103598Z       at Context.<anonymous> (tests.ts:226:30)

**Comment by anthony-c-martin** [comment]
> @anthony-c-martin how can i address these errors?

From the message, it looks like there are some places which are still referencing `https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression` (see my comment https://github.com/Azure/azure-resource-manager-schemas/pull/1524#issuecomment-785401786)

**Comment by grizzlytheodore** [comment]
moved to https://github.com/Azure/azure-resource-manager-schemas/pull/1529

**Comment by grizzlytheodore** [comment]
@anthony-c-martin  ah it was confusing me since there were to Compute.Extension files. thank you for your help

**Review by anthony-c-martin — APPROVED** [review]
LGTM!

**Pull request merged**